Stripe/PublicHeaders/STPBankAccount.h

```diff
@@ -10,85 +10,108 @@
 
 #import "STPBankAccountParams.h"
 #import "STPAPIResponseDecodable.h"
+#import "STPSourceProtocol.h"
 
+NS_ASSUME_NONNULL_BEGIN
 
 /**
  Possible validation states for a bank account.
-
- - STPBankAccountStatusNew: The account has had no activity or validation performed
- - STPBankAccountStatusValidated: Stripe has determined this bank account exists.
- - STPBankAccountStatusVerified: Bank account verification has succeeded.
- - STPBankAccountStatusVerificationFailed: Verification for this bank account has failed.
- - STPBankAccountStatusErrored: A transfer sent to this bank account has failed.
  */
 typedef NS_ENUM(NSInteger, STPBankAccountStatus) {
+    /**
+     The account has had no activity or validation performed
+     */
     STPBankAccountStatusNew,
+
+    /**
+     Stripe has determined this bank account exists.
+     */
     STPBankAccountStatusValidated,
+
+    /**
+     Bank account verification has succeeded.
+     */
     STPBankAccountStatusVerified,
+
+    /**
+     Verification for this bank account has failed.
+     */
     STPBankAccountStatusVerificationFailed,
+
+    /**
+     A transfer sent to this bank account has failed.
+     */
     STPBankAccountStatusErrored,
 };
 
 /**
- Representation of a user's bank account details that have been tokenized with the Stripe API
+ Representation of a user's bank account details that have been tokenized with
+ the Stripe API.
 
  @see https://stripe.com/docs/api#bank_accounts
  */
-@interface STPBankAccount : STPBankAccountParams<STPAPIResponseDecodable>
+@interface STPBankAccount : NSObject<STPAPIResponseDecodable, STPSourceProtocol>
 
 /**
- The last 4 digits of the bank account's account number.
+ You cannot directly instantiate an `STPBankAccount`. You should only use one 
+ that has been returned from an `STPAPIClient` callback.
  */
-- (nonnull NSString *)last4;
+- (nonnull instancetype) init __attribute__((unavailable("You cannot directly instantiate an STPBankAccount. You should only use one that has been returned from an STPAPIClient callback.")));
```

Comment:
No, I think I just copy and pasted from somewhere else that had it wrong. I'll fix both things.